neuraxle/base.py

```diff
@@ -1240,6 +1237,7 @@ def setup(self) -> BaseStep:
         """
         BaseStep.setup(self)
         self.wrapped.setup()
+        self.is_initialized = True
```

Comment:
I wonder why this is here. The `BaseStep.setup(self)` should already take care of setting this to True. Well it looks ok.